install/migrations/update_9.5.x_to_10.0.0/ticket.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2015-2022 Teclib' and contributors.
+ *
+ * http://glpi-project.org
+ *
+ * based on GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * GLPI is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+
+/**
+ * @var DB $DB
+ * @var Migration $migration
+ */
+
+/* Remove global_validation field from templates (should not be defined manually). */
+foreach (['glpi_tickettemplatehiddenfields', 'glpi_tickettemplatemandatoryfields', 'glpi_tickettemplatepredefinedfields'] as $table) {
```

Comment:
as the field can still be displayed (as readonly) in the form, i think it's ok to keep the possibility to add it in glpi_tickettemplatehiddenfields

Reply:
since this is a computed only field, I do not see any benefit of that

Reply:
To not display at all the field in the form

Reply:
oh, ok indeed

Reply:
Problem is that the ITILTemplate logic is not made to have different list of fields for hidden/predefined/mandatory sections. So if this field have to be available in hidden fields only, we have to produce some code to handle this.

Reply:
My bad, there is a logic for this. See 18f51b86f16897b0e5f7042f79d20e99c262ed31.